stratum/lib/utils.cc

```diff
@@ -236,4 +237,16 @@ size_t ProtoHash(const google::protobuf::Message& m) {
   return string_hasher(s);
 }
 
+std::string CxxDemangle(const char* mangled) {
```

Comment:
```suggestion
std::string CxaDemangle(const char* mangled) {
```

We might call it just `Demangle`.